server/pulp/plugins/loader/api.py

```diff
@@ -432,15 +432,16 @@ def load_content_types(types_dir=_TYPES_DIR, dry_run=False, drop_indices=False):
 
     # to handle node.json only
     descriptors = _load_type_descriptors(types_dir)
-    definitions = parser.parse(descriptors)
+    old_definitions = parser.parse(descriptors)
```

Comment:
let's make this variable name better.   s/old_definitions/non_mongoengine_definitions/